bulletin-ui/.gitignore

```diff
@@ -0,0 +1,27 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
```

Comment:
You have now two `.gitignore` files :D Also put all the stuff out side from `bulletin-ui` folder :D